sdk/storage/Azure.Storage.Common/src/Shared/Constants.cs

```diff
@@ -714,5 +714,11 @@ internal static class ServerTimeout
             internal const string HttpMessagePropertyKey = "Azure.Storage.StorageServerTimeoutPolicy.Timeout";
             internal const string QueryParameterKey = "timeout";
         }
+
+        internal static class CopyHttpAuthorization
+        {
+            internal static readonly string[] Scopes = { "https://storage.azure.com/.default" };
```

Comment:
Don't think we actually need `Scopes` anymore since we are getting a token from the Blob client?

Reply:
We actually still need this, but I guess I would move it to the `DataMovementConstants` instead since it's only used by the `DataMovement` package

Reply:
Oh wait I'm wrong. We just need this in general, since it's used [here](https://github.com/Azure/azure-sdk-for-net/pull/36199/files#diff-5abef2aa6e122b00cae3a7ef4d9ccfce5d74df942ba64620d8d5a4ce7430e877)